docs/DeveloperGuide.md

```diff
@@ -262,29 +262,50 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | Financial Advisor | add contacts of my clients                                                         | keep a record of my clients' contact information.                                |
+| `* * *`  | Financial Advisor | find contacts of my clients                                                        | find information on a specific client.                                          |
+| `* * *`  | Financial Advisor | update client's contact information                                                | not need to delete and create new contact information.                           |
+| `* * *`  | Financial Advisor | remove contact information                                                         | reduce clutter in contact list with clients I have severed ties with.           |
+| `* * *`  | Financial Advisor | attach a meeting date and time to my client contact                                | know the next meeting plan with a specific client                               |
+| `* * *`  | Financial Advisor | filter contact list by tags                                                        | update the other parties under the same plan if they were not present in the meeting |
+| `* * *`  | Financial Advisor | delete a scheduled meeting                                                         | keep my schedule up-to-date                                                     |
+| `* * *`  | Financial Advisor | group my clients into different groupings according to financial plans             | easily find target clients                                                      |
+| `* *`    | Financial Advisor | be able to view all my meetings for the day                                       | be prepared for my meetings of the day                                          |
+| `* *`    | Financial Advisor | filter contact list by meeting details                                             | find out who I'm meeting                                                        |
+| `* *`    | Financial Advisor | modify a scheduled meeting's details                                               | keep up-to-date with the meeting's details                                      |
+| `* *`    | Financial Advisor | shift clients into different groups                                                | reorganize in the event of changes                                              |
+| `* *`    | Financial Advisor | be able to add simple notes to my client contact information                       | know their financial goals to prepare me for my next meeting with them.         |
+| `*`      | New user          | import contact information in bulk to FINDvisor                                    | save time and ensure no client is overlooked.                                   |
```

Comment:
I think we should remove the punctuation at the end to keep it consistent? Just a nit.